helper/init.go

```diff
@@ -0,0 +1,239 @@
+package lib
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+
+	"text/template"
+
+	"github.com/resumic/schema/docs/schema"
+	"github.com/tidwall/gjson"
+)
+
+//InitResume initializes the resume.json file
+func InitResume() {
+	var conf, doc, tmpl string
+	flag.StringVar(&conf, "conf", "../schema/schema.json", "Schema json file")
+	flag.StringVar(&tmpl, "tmpl", "resume.tmpl", "Template file")
+	flag.StringVar(&doc, "doc", "resume.json", "Output json file")
+	config, err := ioutil.ReadFile(conf)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	f, err := os.OpenFile(tmpl, os.O_CREATE|os.O_WRONLY, 0666)
+	if err != nil {
+		log.Fatal(err)
+	}
+	f.WriteString("{")
+	value := gjson.GetBytes(config, "properties")
+	j := 0
+	value.ForEach(func(key, value gjson.Result) bool {
+		if j != 0 {
+			f.WriteString("\n ,")
+		}
+		j++
+		_, err = f.WriteString("\"" + key.String() + "\":") //write name of sections
+		if err != nil {
+			log.Fatal(err)
+		}
+		if (isArray(getType(value.String()))) == true {
+			_, err = f.WriteString("[") //write for type array
+			if err != nil {
+				log.Fatal(err)
+			}
+		}
+		_, err = f.WriteString("{\n")
+		if err != nil {
+			log.Fatal(err)
+		}
+		fmt.Println(key.String())
+		result := gjson.GetBytes(config, "properties."+key.String())
+		result.ForEach(func(key1, value gjson.Result) bool {
+			if key1.String() == "properties" {
+				i := 0
+				result1 := gjson.GetBytes(config, "properties."+key.String()+"."+key1.String())
+				result1.ForEach(func(key2, value gjson.Result) bool {
+					if i != 0 {
+						f.WriteString(",")
+					}
+					i++
+					f.WriteString("\"" + key2.String() + "\":")
+					if isArray(getType(value.String())) == true {
+						f.WriteString("[")
+					}
+					val := getFormat(value.String())
+					println(val)
```

Comment:
I don't think that we need to print values to the user. Or if they are needed, change this line and add fmt package.

Reply:
@erbesharat yes right, not needed anymore.